docs/docs-components/COMPONENT_DATA.js

```diff
@@ -1082,6 +1082,56 @@ const GENERAL_COMPONENT_LIST: $ReadOnlyArray<ListItemType> = [
       figma: null,
     },
   },
+  {
+    svg: <IconButton />,
```

Comment:
We should ask Kate to prep a new SVG here

Reply:
What is the process to address that?

Reply:
Probably ask Kate Halvorsen